### PR TITLE
Release/0.9.5

### PR DESCRIPTION
**Pull request by csadorf**

### CODE REVIEW

**Comment by csadorf** [comment]
Outstanding tasks:
[ ] Clean up branch history before merge.
[ ] Setup Circle-CI PyPI release step.
[ ] Bump version numbers.

**Comment by csadorf** [comment]
@glotzerlab/signac-maintainers I've setup the CircleCI deployment pipeline. This release is ready for the version bump. I'd appreciate your approval.

**Review by vyasr — APPROVED** [review]

**Review by bdice — APPROVED** [review]

**Pull request merged**
